pom.xml

```
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.46</version>
+        <version>4.74</version>
```

Optional improvement to use most recent plugin parent pom release.
```suggestion
        <version>4.76</version>
```